### PR TITLE
test(webpack): update config for styling

### PR DESCRIPTION
**Pull request by DlgSHi**
### Proposed behaviour

- fix for the `webpack.config.js` file
- amendments for the `px` value
- slight refactor tests to pass locally and on CI
- some `px` value were changed due to changed visual part (icons/fonts)

### Current behaviour

- no `carbon` icons/fonts were shown when Cypress was running
- all tests were passing on CI but there were a few that are failing locally.

### Checklist

- [x] Commits follow our style guide
- [ ] Related issues linked in commit messages if required
- [ ] Screenshots are included in the PR if useful
- [ ] All themes are supported if required
- [ ] Unit tests added or updated if required
- [x] Cypress automation tests added or updated if required
- [ ] Storybook added or updated if required
- [ ] Translations added or updated (including creating or amending translation keys table in storybook) if required
- [ ] Typescript `d.ts` file added or updated if required
- [ ] Related docs have been updated if required

#### QA

- [ ] Tested in CodeSandbox/storybook
- [ ] Add new Cypress test coverage if required
- [ ] Carbon implementation matches Design System/designs
- [ ] UI Tests GitHub check reviewed if required
- [ ] Check that `carbon` icons/fonts are shown when Cypress running
- [ ] Check that all specs are passing locally as well

### CODE REVIEW

File: src/components/content/content.test.js [hunk 1]
```diff
@@ -72,7 +70,9 @@ context("Tests for Content component", () => {
       "should check %s% width for Content component",
```
Comment (divyajindel) [inline src/components/content/content.test.js:70]:
```suggestion
      "should check %s width for Content component",
```

**Review by divyajindel — DISMISSED** [review]

File: src/components/pod/pod.test.js [hunk 2]
```diff
@@ -65,13 +68,15 @@ const SoftDeletePodWithChildren = () => (
...
+      [true, 1, "solid", "rgb(204, 214, 219)"],
+      [false, 0, "none", "rgba(0, 0, 0, 0.9)"],
     ])(
       "should check when border is %s the border value is %s for Pod component ",
-      (boolVal, border) => {
+      (boolVal, px, style, color) => {
         CypressMountWithProviders(<PodComponent border={boolVal} />);
-        podBlock().should("have.css", "border", border);
+        podBlock().then((elem) => {
+          checkOutlineCss(elem, 1, "border", style, color);
```
Comment (stephenogorman) [inline src/components/pod/pod.test.js:78]: Should this be:
          checkOutlineCss(elem, px, "border", style, color);
?

File: cypress/support/component-helper/common-steps.js [hunk 3]
```diff
@@ -23,11 +23,28 @@ export const parseToIntElement = (elem) => {
   return parseInt(inputString);
 };
 
-export const checkGoldenOutline = (elem, outlineWidthPx = 3) => {
-  const outlineWidth = parseToIntElement(elem.css("outline-width"));
-  expect(elem.css("outline-color")).to.equals("rgb(255, 181, 0)");
-  expect(elem.css("outline-style")).to.equals("solid");
-  expect(outlineWidth).to.be.within(outlineWidthPx - 1, outlineWidth + 1);
+export const checkGoldenOutline = (
```
Comment (edleeks87) [inline cypress/support/component-helper/common-steps.js:26]: If this is no longer only checking `outline` maybe something more generic would be better wrt to naming of the function and args:

```
  export const checkFocusStyling = (elem, focusStyleWidthPx = 3, focustStyleProperty) => ...
```

File: cypress/support/component-helper/common-steps.js [hunk 4]
```diff
@@ -23,11 +23,28 @@ export const parseToIntElement = (elem) => {
...
+  outlineWidthPx = 3,
+  outline = "outline"
+) => {
+  const outlineWidth = parseToIntElement(elem.css(`${outline}-width`));
+  expect(elem.css(`${outline}-color`)).to.equals("rgb(255, 181, 0)");
+  expect(elem.css(`${outline}-style`)).to.equals("solid");
+  expect(outlineWidth).to.be.within(outlineWidthPx - 1, outlineWidthPx + 1);
+};
+
+export const checkOutlineCss = (
```
Comment (edleeks87) [inline cypress/support/component-helper/common-steps.js:37]: Similar comment wrt naming it something more generic. Neither is essential though, happy to keep as is if you prefer it current naming

**Review by edleeks87 — COMMENTED** [review]
Changes to webpack config look okay to me

File: cypress/support/component-helper/common-steps.js (hunk 3 above)
Comment (DlgSHi) [reply cypress/support/component-helper/common-steps.js:26]: this function actually checks only the focus (golden outline) so I would leave it as it is

File: cypress/support/component-helper/common-steps.js (hunk 4 above)
Comment (DlgSHi) [reply cypress/support/component-helper/common-steps.js:37]: same here, this checks only the outline (border) for different components with diff styling / colors / width - so I think it is ok as well :)

**Review by edleeks87 — APPROVED** [review]

**Pull request merged**